IterableAppExtensionsTests/IterableAppExtensionsTests.m

```diff
@@ -252,6 +257,49 @@ - (void)testPushTextInputButton {
     [self waitForExpectations:@[expectation] timeout:IterableNotificationCenterExpectationTimeout];
 }
 
+- (void)testPushTextInputForegroundButton {
+    UNMutableNotificationContent *content = [[UNMutableNotificationContent alloc] init];
+    content.userInfo = @{
+                         @"itbl" : @{
+                                 @"messageId": [[NSUUID UUID] UUIDString],
+                                 @"actionButtons": @[@{
+                                                         @"identifier": @"textInputButton",
+                                                         @"title": @"Text Input",
+                                                         @"buttonType": @"textInput",
+                                                         @"openApp": @YES,
+                                                         @"action": @{
+                                                                 
+                                                                 }
+                                                         }]
+                                 }
+                         };
+    UNNotificationRequest *request = [UNNotificationRequest requestWithIdentifier:@"request" content:content trigger:nil];
+    
+    XCTestExpectation *expectation = [[XCTestExpectation alloc] initWithDescription:@"contentHandler is called"];
+    
+    [self.extension didReceiveNotificationRequest:request withContentHandler:^(UNNotificationContent *contentToDeliver) {
+        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, IterableNotificationCenterRequestDelay * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
+            UNUserNotificationCenter* center = [UNUserNotificationCenter currentNotificationCenter];
+            [center getNotificationCategoriesWithCompletionHandler:^(NSSet<UNNotificationCategory *> * _Nonnull categories) {
+                UNNotificationCategory *createdCategory = nil;
+                for (UNNotificationCategory *category in categories) {
+                    if ([category.identifier isEqualToString:content.userInfo[@"itbl"][@"messageId"]])
+                        createdCategory = category;
```

Comment:
Just a suggestion and not necessary for PR approval. I like open/close brackets after single line if statements even though it is not necessary.

Reply:
Right! I'm so used to it I don't notice.. I'll see if there are any style checks in AppCode for that.